### DISCUSSION TITLE
top genes expressed in a region

### DISCUSSION THREAD

**Discussion by sokratiag** [BayraktarLab/cell2location#13] [Q&A] [answered]
Hi,

First of all thank you for this useful tool!  

I have one question regarding the features. In the colocated analysis, is it possible to extract the top genes that are expressed in the 'fact_x' regions? If not in the colocated analysis, is there generally a way to acquire this info in c2l?

Thanks!

**Comment by vitkl** [comment, marked as answer]
Hi @sokratiag 

Glad that you find it useful. Finding genes linked to a certain region is not a problem cell2location aims to address. However, you can use any NB regression model (e.g. DESeq2), providing cell type combinations across locations (from NMF analysis) as a covariate. You can do the same for discrete regions defined by clustering (e.g. our regression model from tutorial 1/3, DESeq2).

We have recently discovered Discussions feature on GitHub so would be better to have your question here.

**Reply by sokratiag** [reply]
Thanks vitkl!
